test/pingtimes.ts

```diff
@@ -126,4 +126,33 @@ describe("Pingtimes", function() {
       modeMatcher(gapsMode);
     }
   });
+
+  describe("backwards compatibility with original TagTime", function() {
+    context("with the default seed", function() {
+      beforeEach(function(){
+        pings = new PingTimes(45 * 60 * 1000, 11193462 , PingTimes.epoch);
+      });
+      it("first ping should be reproduced", function() {
+        const time = PingTimes.epoch+1;
```

Comment:
this shadows the `time` defined higher up, so perhaps rename it and its use to `t`, here and in the following three functions.